.github/copilot-instructions.md

```diff
@@ -0,0 +1,42 @@
+## Test Suggestions
+
+When converting a PR from WIP to publish, provide a concise test plan.
+
+- **Select Key Test Cases:** List only the essential test cases to validate the PR. Use pipe '|' to separate names (e.g., test1|test2|test3).
```

Comment:
When the change is not related specific test cases, choose the test cases can test the change.